openeogeotrellis/geopysparkdatacube.py

```diff
@@ -512,12 +512,26 @@ def tilefunction(metadata: Metadata, openeo_metadata: GeopysparkCubeMetadata,
             _log.info(f"apply_tiles tilefunction result dims: {result_array.dims}")
             return (key,Tile(result_array.values, geotrellis_tile[1].cell_type,geotrellis_tile[1].no_data_value))
 
-        def rdd_function(openeo_metadata: GeopysparkCubeMetadata, rdd):
-            numpy_rdd = rdd.convert_data_type(CellType.FLOAT32).to_numpy_rdd().map(
-                log_memory(partial(tilefunction, rdd.layer_metadata, openeo_metadata))
-            )
-            metadata = GeopysparkDataCube._transform_metadata(rdd.layer_metadata, cellType=CellType.FLOAT32)
-            return gps.TiledRasterLayer.from_numpy_rdd(rdd.layer_type, numpy_rdd, metadata)
+        if runtime == 'PythonOffheap':
```

Comment:
Maybe the name of the runtime can point more to the actual environment, like 'Python-Jep', as that's probably what's going to be the most noticable difference for the user.